diff -rNu why3-0.85.orig/share/provers-detection-data.conf why3-0.85/share/provers-detection-data.conf
--- why3-0.85.orig/share/provers-detection-data.conf	2014-09-17 09:15:34.000000000 +0200
+++ why3-0.85/share/provers-detection-data.conf	2015-01-04 21:12:48.000000000 +0100
@@ -5,6 +5,7 @@
 version_switch = "-version"
 version_regexp = "\\([0-9.]+\\(-dev\\)?\\)"
 version_ok  = "0.95.2"
+version_ok  = "0.95.2.20140901"
 command = "%l/why3-cpulimit %t %m -s %e -timelimit %t %f"
 driver = "drivers/alt_ergo.drv"
 editor = "altgr-ergo"
