diff -rNu why3-0.85.orig/share/provers-detection-data.conf why3-0.85/share/provers-detection-data.conf
--- why3-0.85.orig/share/provers-detection-data.conf	2014-09-17 09:15:34.000000000 +0200
+++ why3-0.85/share/provers-detection-data.conf	2015-01-04 21:11:47.000000000 +0100
@@ -297,6 +297,16 @@
 [ATP z3]
 name = "Z3"
 exec = "z3"
+exec = "z3-4.3.2"
+version_switch = "-version"
+version_regexp = "Z3 version \\([^ \n\r]+\\)"
+version_ok  = "4.3.2"
+driver = "drivers/z3.drv"
+command = "%l/why3-cpulimit %t %m -s %e -T:%t -smt2 %f"
+
+[ATP z3]
+name = "Z3"
+exec = "z3"
 exec = "z3-4.3.1"
 exec = "z3-4.3.0"
 exec = "z3-4.2"
