diff -rNu why3-0.85.orig/Makefile.in why3-0.85/Makefile.in
--- why3-0.85.orig/Makefile.in	2014-09-17 09:15:26.000000000 +0200
+++ why3-0.85/Makefile.in	2015-01-27 15:00:08.641286500 +0100
@@ -1198,7 +1198,7 @@
 OCAMLLIBS_CMO = $(addsuffix .cmo, $(OCAMLLIBS_MODULES))
 OCAMLLIBS_CMX = $(addsuffix .cmx, $(OCAMLLIBS_MODULES))
 
-$(OCAMLLIBS_DEP): DEPFLAGS += -I src/util -I lib/ocaml @BIGINTINCLUDE@
+$(OCAMLLIBS_DEP): DEPFLAGS += -I src/util -I lib/ocaml
 $(OCAMLLIBS_CMO) $(OCAMLLIBS_CMX): INCLUDES += -I src/util -I lib/ocaml @BIGINTINCLUDE@
 $(OCAMLLIBS_CMX): OFLAGS += -for-pack Why3extract
 
